homeassistant/components/iaqualink/__init__.py

```diff
@@ -210,3 +210,24 @@ def should_poll(self) -> bool:
     def unique_id(self) -> str:
         """Return a unique identifier for this entity."""
         return f"{self.dev.system.serial}_{self.dev.name}"
+
+    @property
+    def assumed_state(self) -> bool:
+        """Return whether the state is based on actual reading from the device."""
+        return self.dev.system.last_run_success is False
+
+    @property
+    def available(self) -> bool:
+        """Return whether the device is available or not."""
+        return self.dev.system.online is True
+
+    @property
+    def device_info(self) -> Dict[str, Any]:
+        """Return the device info."""
+        return {
+            "identifiers": {(DOMAIN, self.unique_id)},
```

Comment:
Will there only be one entity per device?

Reply:
I'm not sure I understand the question. Do you have an example where it's not the case?

Reply:
We're using the entity unique_id as identifier. If there would be more than one entity per device the mapping wouldn't be correct. This I'd should be unique per device and not unique per entity.

Reply:
Was looking for examples where you'd have more than one entity for a given device.

Reply:
I think we're good here in any case. There's a 1:1 mapping between entities and devices/controls on the iaqualink side.